src/hospitals/hospitals.resolver.ts

```diff
@@ -0,0 +1,53 @@
+import { Resolver, Query, Mutation, Args } from '@nestjs/graphql';
+import { HospitalsService } from './hospitals.service';
+import { CreateHospitalInput } from './dto/create-hospital.input';
+import { UseGuards } from '@nestjs/common';
+import { AdminGuard } from 'src/guards/admin.guard';
+import { EditHospitalInput } from './dto/edit-hospital.input';
+
+@Resolver('Hospital')
+export class HospitalsResolver {
+  constructor(private readonly hospitalsService: HospitalsService) {}
+
+  @Mutation('createHospital')
+  @UseGuards(AdminGuard)
```

Comment:
Guard can be applied to the whole controller as it used for all methods.

Reply:
There is no place where I can use it, because in every module I need to use several guards or don't use them at all